plugins/runtime/ts/runtime.init.ts

```diff
@@ -0,0 +1,28 @@
+/// <reference path="../../jmx/ts/workspace.ts"/>
+/// <reference path="runtime.service.ts"/>
+
+namespace Runtime {
+
+  export function RuntimeInit($rootScope: ng.IScope, viewRegistry, helpRegistry,
+    workspace: Jmx.Workspace, runtimeService: RuntimeService) {
+    'ngInject';
+    
+    viewRegistry['runtime'] = 'plugins/runtime/html/layout.html';
+
+    helpRegistry.addUserDoc('runtime', 'plugins/runtime/doc/help.md');
+
+    const unsubscribe = $rootScope.$on('jmxTreeUpdated', () => {
+      unsubscribe();
+      const tabs = runtimeService.getTabs();
+      workspace.topLevelTabs.push({
+        id: "runtime",
+        content: "Runtime",
+        title: "Runtie",
+        isValid: () => tabs.length > 0,
+        href: () => tabs[0].path,
+        isActive: (workspace: Jmx.Workspace) => workspace.isLinkActive("diagnostics")
```

Comment:
You should check the Runtime link is active, not the Diagnostics link. Copy & paste error? :-)

Reply:
how on earth did you guess that? My bad. Nice catch